charts/incubator/openldap_argon2/questions.yaml

```diff
@@ -0,0 +1,323 @@
+            required: true
+            default: "example.org"
+        - variable: LDAP_READONLY_USER
+          label: "LDAP_READONLY_USER"
+          schema:
+            type: boolean
+            default: false
+        - variable: LDAP_RFC2307BIS_SCHEMA
+          label: "LDAP_RFC2307BIS_SCHEMA"
+          schema:
+            type: boolean
+            default: false
+        - variable: LDAP_BACKEND
+          label: "LDAP_BACKEND"
+          schema:
+            type: string
+            required: true
+            default: "mdb"
+        - variable: LDAP_TLS
+          label: "LDAP_TLS"
+          schema:
+            type: boolean
+            default: true
+        - variable: LDAP_TLS_ENFORCE
+          label: "LDAP_TLS_ENFORCE"
+          schema:
+            type: boolean
+            default: false
+        - variable: LDAP_TLS_VERIFY_CLIENT
+          label: "LDAP_TLS_VERIFY_CLIENT"
+          schema:
+            type: string
+            required: true
+            default: "never"
+        - variable: LDAP_TLS_PROTOCOL_MIN
+          label: "LDAP_TLS_PROTOCOL_MIN"
+          schema:
+            type: string
+            required: true
+            default: "3.0"
+        - variable: LDAP_TLS_CIPHER_SUITE
+          label: "LDAP_TLS_CIPHER_SUITE"
+          schema:
+            type: string
+            required: true
+            default: "NORMAL"
+        - variable: LDAP_TLS_REQCERT
+          label: "LDAP_TLS_REQCERT"
+          schema:
+            type: string
+            required: true
+            default: "never"
+        - variable: CONTAINER_LOG_LEVEL
+          label: "CONTAINER_LOG_LEVEL"
+          schema:
+            type: int
+            required: true
+            default: 4
+        - variable: KEEP_EXISTING_CONFIG
+          label: "KEEP_EXISTING_CONFIG"
+          schema:
+            type: boolean
+            default: false
+        - variable: LDAP_REMOVE_CONFIG_AFTER_SETUP
+          label: "LDAP_REMOVE_CONFIG_AFTER_SETUP"
+          schema:
+            type: boolean
+            default: true
+        - variable: LDAP_SSL_HELPER_PREFIX
+          label: "LDAP_SSL_HELPER_PREFIX"
+          schema:
+            type: string
+            required: true
+            default: "ldap"
+        - variable: LDAP_BASE_DN
+          label: "LDAP_BASE_DN"
+          schema:
+            type: string
+            required: true
+            default: ""
+  - variable: customLdifFiles
+    group: "custom ldif files"
+    label: "custom ldif"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: 50-argon2-module.ldif
+          label: "argon2 module ldif"
+          schema:
+            type: string
+            required: false
+            default: |-
+              # Add in argon2 hashing support
+              dn: cn=module{0},cn=config
+              changetype: modify
+              add: olcModuleLoad
+              olcModuleLoad: {0}pw-argon2
```

Comment:
there is no multiline support in scale UI

Reply:
Please correct me if I'm wrong, but it seems that these values are not adjustable by questions.yaml for now.  And they will have to move to [ix_values.yaml]https://truecharts.org/docs/manual/development/values-yaml#ix_valuesyaml) and this will remain a hard fork from the base openldap app for now?